content/kubermatic/master/architecture/compatibility/OS-support-matrix/_index.en.md

```diff
@@ -9,13 +9,20 @@ weight = 3
 
 Kubermatic supports a multitude of operating systems. One of the unique features of Kubermatic Kubernetes Platform is the possibility to recombine different machine deployments and operating systems even in one cluster. If you intend to use more than one type of OS and/or CRI in one K8s cluster, please consult with Kubermatic Support.
 
-The operating systems below are currently supported by Kubermatic:
+The operating systems below are currently supported on the given cloud providers by Kubermatic:
 
-* RHEL beginning with 8.0 (support is cloud provider-specific)
-* CentOS beginning with 7.4
-* Ubuntu LTS beginning with 18.04
-* Flatcar (Stable channel)
-* SLES beginning with 15.0 (only on AWS)
-* Ubuntu 20.04
+|   | Ubuntu | CentOS | Flatcar | RHEL | SLES | Amazon Linux 2 | Rocky Linux |
```

Comment:
Should we maybe keep the version information (e.g. that we only support CentOS 7, Ubuntu 20.04, Rocky Linux 8, etc)? It's not obvious now that we don't support 22.04 yet.

Reply:
That's a really important point, let me update this.